src/WheelsAndGoods.Core/Models/Paginations/Responses/PaginationOrderResponse.cs

```diff
@@ -0,0 +1,10 @@
+﻿using WheelsAndGoods.Core.Models.Entities;
+
+namespace WheelsAndGoods.Core.Models.Paginations.Responses;
+
+public class PaginationOrderResponse
```

Comment:
We will be able to reuse this model in the future if it will be generic `PaginationOrderResponse<TModel>` then Data will be `IReadOnlyCollection<TModel>` .